blacklisted_websites.txt

```diff
@@ -1071,3 +1071,22 @@ healthflyup\.com
 supplementschoice\.com
 lumagenexuk\.co\.uk
 garciniacambogialean\.com
+ nutritionfit\.org
+oralhealthplus\.com
+drhelpnutrition\.org
+supplementlab\.org
+supplementstest\.org
+megacleanseradvice\.com
+worldmuscleking\.com
+quicksupplementfact\.com
+maleenhancementshop\.info
+gomusclebuilding\.com
+healthyorder\.org
```

Comment:
The majority of these are already pattern-matched, and some are already blacklisted (like this one). This will need the be reviewed and pruned down before merging.